polyply/tests/test_lib_files.py

```diff
@@ -85,7 +85,8 @@ def assert_equal_blocks(block1, block2):
                     if _interaction_equal(ref_inter, new_inter, inter_type):
                         break
```

Comment:
Doesn't this test just the first interaction?

Reply:
yes it scans new interactions until it finds one match in ref_interactions. This complicated syntax has to do with multiple dihedrals and these kind of things.